step.sh

```diff
@@ -452,6 +450,16 @@ for a_app_dsym in $(find "${archive_dsyms_folder}" -type d -name "*.app.dSYM") ;
 done
 unset IFS
 
+dsym_zip_path="${output_dir}/${scheme}.dSYM.zip"
+
+IFS=$'\n'
+for a_app_dsym in $(find "${archive_dsyms_folder}" -type d -name "*.app.dSYM") ; do
```

Comment:
I'm not convinced that this should be handled in a second loop, would be better to handle in the first loop